game/options.rpy

```diff
@@ -23,7 +23,7 @@ define gui.show_name = False
 
 ## The version of the game.
 
-define config.version = "0.0.2"
+define config.version = "0.0.3"
```

Comment:
We should only change this when we're going to distribute a release, and this should be in its own commit/pr

Reply:
Got it